lib/opal/nodes/top.rb

```diff
@@ -76,8 +76,13 @@ def compile_irb_vars
         end
       end
 
+      def magic_comments_helpers
+        @magic_comments_helpers ||=
+          compiler.magic_comments[:helpers].to_s.split(',').map(&:strip)
+      end
+
       def add_used_helpers
-        helpers = compiler.helpers.to_a
+        helpers = compiler.helpers.to_a + magic_comments_helpers
```

Comment:
what happens if the file declares the helper and the code implicitly declares it via `helper` DSL in the compiler node? It worked well before (because `compiler.helpers` is a set), but I guess now it can declare them twice.